python/tank/pipelineconfig.py

```diff
@@ -69,28 +70,44 @@ def __init__(self, pipeline_configuration_path, descriptor=None):
 
         # and get the version of the API currently in memory
         current_api_version = pipelineconfig_utils.get_currently_running_api_version()
-        
+
```

Comment:
I'm not sure what I did to trigger all these removals of whitespace. If you think i should go back and re-do my changes, let me know.

Reply:
Please, keep them in :)

Reply:
pycharm is doing it and you are hitting an old piece of toolkit. good to get it standardized! :)